Apps/Inc/CLI.h

```diff
@@ -69,6 +71,9 @@
 // Useful values
 #define NUM_TEMP_SENSORS_LAST_MINION    14
 
+//For pending/posting CLI resource semaphores/mutexes
+enum CLI_Resource{SafetyCheck, MinionsASIC, VoltageBuffer, TemperatureBuffer, OpenWireBuffer, AmperesData};
```

Comment:
Can you make these all caps, so it is easy to see that these are constants when we are reading the code? Also, can you `typedef` this to something descriptive?